src/Setup.php

```diff
@@ -437,14 +392,14 @@ public function bcgov_block_theme_wp_headers( $headers ) {
         $csp         = '';
         $csp_initial = [
             'upgrade-insecure-requests',
-            'default-src'     => "'self' " . esc_attr( get_option( 'custom_csp_defaultsrc' ) ),
-            'script-src'      => "'self' 'unsafe-inline' 'unsafe-eval' " . esc_attr( get_option( 'custom_csp_scriptsrc' ) ),
-            'style-src'       => "'self' 'unsafe-inline' " . esc_attr( get_option( 'custom_csp_stylesrc' ) ),
-            'connect-src'     => "'self' " . esc_attr( get_option( 'custom_csp_connectsrc' ) ),
-            'font-src'        => "'self' 'unsafe-inline' data: " . esc_attr( get_option( 'custom_csp_fontsrc' ) ),
-            'img-src'         => "'self'  data: " . esc_attr( get_option( 'custom_csp_imgsrc' ) ),
-            'media-src'       => "'self' 'unsafe-inline' " . esc_attr( get_option( 'custom_csp_mediasrc' ) ),
-            'frame-src'       => "'self' " . esc_attr( get_option( 'custom_csp_framesrc' ) ),
+            'default-src'     => "'self' gov.bc.ca *.gov.bc.ca data: *.twitter.com *.twimg.com " . ( get_option( 'custom_csp_defaultsrc' ) ? esc_attr( get_option( 'custom_csp_defaultsrc' ) ) : '' ),
```

Comment:
why not equate these all to variables, i wouldn't trust the `WordPress` handles this and caches, it, as what you have done actually calls everything 2x

Reply:
Ok, will do.